notebooks/pymc4_design_guide.ipynb

```diff
@@ -11,7 +11,7 @@
    "cell_type": "markdown",
```

Comment:
<p>"Let's fist" should be "Let's first"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/pymc4/pull/150/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Reply:
Hm getting confused if this was already fixed, here's what I see on ReviewNB 
![image](https://user-images.githubusercontent.com/7213793/66271901-36565f00-e818-11e9-8c8a-c90f464acfe1.png)

Reply:
Thanks - I missed this one.